packages/landing/src/widgets/landing/subscription-section.tsx

```diff
@@ -60,7 +61,9 @@ export function SubscriptionSection() {
             <input value={email} className="input" onChange={handleEmailChange} />
           </InputBox>
 
-          <Button onClick={handleSubscribe}>Subscribe</Button>
+          <Button onClick={(e: any) => handleSubscribe(e)} type="submit">
```

Comment:
It seems with `type="submit"` this button already calls form's onSubmit event, isn't it? So you can remove the onClick handler.